framework/test/mocha/functional/ws/transport/transport.js

```diff
@@ -127,27 +127,23 @@ describe('WS transport', () => {
 				const blockRes = await blocksEndpoint.makeRequest({ height: 2 }, 200);
 				const blockId = blockRes.body.data[0].id;
 				const { data } = await p2p.request({
-					procedure: 'blocks',
-					data: { lastBlockId: blockId },
+					procedure: 'getBlocksFromId',
+					data: { blockId },
 				});
-				expect(data)
-					.to.have.property('blocks')
-					.to.be.an('array');
+				expect(data).to.be.an('array');
 			});
 
-			it('using valid headers should be ok', async () => {
+			it('using a valid payload should be ok', async () => {
```

Comment:
I see it was using this `it using a valid ...` format but would be nice if we could refactor to: `it( 'should use a valid payload')`